apps/crn-server/test/controllers/research-outputs.test.ts

```diff
@@ -190,6 +190,10 @@ describe('ResearchOutputs controller', () => {
       researchOutputRequest.link = 'https://some-test-link.com';
 
       test('Should throw error when a research output with the same type and title already exists', async () => {
+        const researchOutputWhitespacesRequest = getResearchOutputCreateData();
```

Comment:
I don't think I like changing the existing test so it works for your case. We should keep the existing one as is because its a valid scenario of when the title is identical and should add an extra one where it's the same but with extra spaces.